oap-server/server-bootstrap/src/main/resources/component-libraries.yml

```diff
@@ -287,6 +287,9 @@ Undertow:
 Finagle:
   id: 85
   languages: Java,Scala
+Mariadb:
```

Comment:
I think you need a logo PR at the rocketbot UI repo.

One question, does `Mariadb` or `Mariadb-JDBC` make more sense? if `Mariadb-JDBC`, then should add mapping to `Component-Server-Mappings:`

Reply:
Ok, I agree. Other JDBC drivers are also using similar naming convention, will refactor it.